src/main/java/goodgenerator/util/StackUtils.java

```diff
@@ -122,4 +127,23 @@ public static FluidStack getTieredFluid(int aTier, int aAmount) {
                 return MyMaterial.shirabon.getMolten(aAmount);
         }
     }
+
+    // === Copied from AE2FC to avoid hard dep ===
+
+    public static IAEFluidStack createAEFluidStack(Fluid fluid) {
```

Comment:
Going to need to need to optional all these on AE2, as you're using the AEApi without a real AE2 dep.

Reply:
Actually GT has hard dep on AE2 now. Removed those optionals